### ISSUE TITLE
Make System.Net.Http nuget package compatible with a "Portable Class Library" Project

### ISSUE THREAD

**Issue by gprossliner** [dotnet/corefx#8259]
When trying to install the System.Net.Http nuget package from nuget.org into a "Portable Class Library Project", the following message is displayed, and the operation is canceled:

"Could not install package 'System.Net.Http 4.0.0'. You are trying to install this package into a project that targets '.NETPortable,Version=v4.5,Profile=Profile78', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author."

The reason of the message is, that there is no "portable-net45+win8+wp8+wpa81" folder in "lib". 

The listed the dependencies for the package in System.Net.Http.nuspec should allow a lib to be included, because all of them are specified in <group targetFramework elements.

**Comment by davidsh** [comment]
cc: @ericstj @stephentoub

**Comment by ericstj** [comment]
This is something that's already been fixed if you are using the RC3 packages.

Try using System.Net.Http 4.1.0-rc3-24104-00 from https://dotnet.myget.org/F/dotnet-core/api/v3/index.json.

If you still repro the issue let us know and I'll reopen this.

**Comment by gprossliner** [comment]
Thank you! It works. Can you tell me when this package will be available on nuget.org?

**Comment by davidsh** [comment]
I believe these fixes are already in RC2 packages which should be pushed out to NuGet.org next week.

**Comment by DYl-iOS** [comment]
@ericstj  
I am new for xamarin. I am working with tamarin studio, facing same issue"Could not install package 'System.Net.Http 4.1.0'. You are trying to install this package into a project that targets 'portable-net45+win+wp80+MonoTouch10+MonoAndroid10+xamarinmac20+xamarintvos10+xamarinwatchos10+xamarinios10', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author."  as you suggested the solution " System.Net.Http 4.1.0-rc3-24104-00 " using this,I am not able to check h"-rc3-24104-00" as I am trying to add System.Net.Http 4.1.0". 

please help me by telling step by step solution.
Thanks in advance

**Comment by DYl-iOS** [comment]
@ericstj I do not need windows support in my project ,  do I need to change "wpa81 " in that case too..
also please tell me how to make this change in profile.
many thanks for your quick reply.

**Comment by ericstj** [comment]
You have a portable library project, open the project properties, under 
Targeting click the `Change...` button.  Uncheck the box that says `Windows Phone Silverlight 8`.

**Comment by DYl-iOS** [comment]
@ericstj  Thanks for your response, but this is not workable solution for me , I solved this issue using steps like>
I added a new .net type Library project <Name:Project.utils> to my solution and then added System.Net.Http package to this new project. 

it successfully added and allow me to use through out the solution. this utils project will be used for all model classes for solution.
after compilation every project converts into .dll library, so i made one library/project to have our HTTP and other Utils functionality at one place and  all rest of the projects will be carrying only Operationg system specific and one Form related code
the reason because of which I am bound to do it is, I have seen sometimes few libraries are not working with xamarin.forms

Thats it .

Thanks 
Ruchi
